python/grass/jupyter/utils.py

```diff
@@ -202,6 +202,28 @@ def get_rendering_size(region, width, height, default_width=600, default_height=
     return (default_width, round(default_width * region_height / region_width))
 
 
+def save_vector(name, geo_json):
+    """
+    Saves the user drawn vector.
+
+    param geojson_filename: name of the geojson file to be saved
+    param name: name with which vector should be saved
+    """
+    import tempfile
+    import json
+
+    try:
+        with tempfile.NamedTemporaryFile(
+            suffix=".geojson", delete=False, mode="w"
+        ) as temp_file:
+            temp_filename = temp_file.name
```

Comment:
When importing, it's saving some junk about style into the database, so I suggest removing properties from json:
```
        for each in geo_json["features"]:
            each["properties"].clear()
```